lib/stitches/valid_mime_type.rb

```diff
@@ -1,19 +1,26 @@
 require_relative 'allowlist_middleware'
 module Stitches
-  # A middleware that requires all API calls to be for versioned JSON.  This means that the Accept
-  # header (available to Rack apps as HTTP_ACCEPT) should be like so:
+  # A middleware that requires all API calls to be for versioned JSON or Protobuf.
+  #
+  # This means that the Accept header (available to Rack apps as HTTP_ACCEPT) should be like so:
   #
   #     application/json; version=1
   #
   # This just checks that you've specified some numeric version.  ApiVersionConstraint should be used
   # to "lock down" the versions you accept.
+  # 
+  # Or in the case of a protobuf encoded payload the header should be like so:
+  #
+  #     application/protobuf
+  #
+  # There isn't an accepted standard for protobuf encoded payloads but this form is common.
   class ValidMimeType < Stitches::AllowlistMiddleware
 
   protected
 
     def do_call(env)
       accept = String(env["HTTP_ACCEPT"])
-      if accept =~ %r{application/json} && accept =~ %r{version=\d+}
+      if (accept =~ %r{application/json} && accept =~ %r{version=\d+}) || accept =~ %r{application/protobuf}
```

Comment:
One version minor nitpicky comment - you might consider using some helper methods here to increase readability:

```ruby
if accept_versioned_json? || accept_protobuf?
  @app.call(env)
else
```

Reply:
that looks nicer however it would actually have to be:
```ruby
accept = String(env["HTTP_ACCEPT"])
if accept_versioned_json?(accept) || accept_protobuf?(accept)
  @app.call(env)
else
```
Does that still look like a worthy improvement.

Reply:
Good point.  I do like it, however the decision is yours!

Reply:
FWIW, I also like the alternate approach for readability.